java/wms-layer-url/src/main/java/com/esri/arcgisruntime/sample/wmslayerurl/MainActivity.java

```diff
@@ -0,0 +1,68 @@
+/* Copyright 2017 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgisruntime.sample.wmslayerurl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+
+import com.esri.arcgisruntime.layers.WmsLayer;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.view.MapView;
+
+public class MainActivity extends AppCompatActivity {
+
+    private MapView mMapView;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        // inflate MapView from layout
+        mMapView = findViewById(R.id.mapView);
+        // create a map with the BasemapType topographic
+        ArcGISMap map = new ArcGISMap(Basemap.Type.IMAGERY, 2.0, 18.0, 3);
+        // set the map to be displayed in this view
+        mMapView.setMap(map);
+
+        // hold a list of uniquely-identifying WMS layer names to display
+        List<String> wmsLayerNames = new ArrayList<>();
+        wmsLayerNames.add("0");
```

Comment:
This looks like an ID rather than a layer 'name' - is this the right value?

Reply:
I've just replicated this from initial .NET implementation, but I can clarify in the next samples scrum

Reply:
Ok, so apparently that is the name of the layer. The WMS service 2 layers with the inspiring names '0' and '1' :P

https://certmapper.cr.usgs.gov/arcgis/services/geology/africa/MapServer/WMSServer?request=GetCapabilities&service=WMS